src/Private/OpenApi.ps1

```diff
@@ -186,33 +354,22 @@ function ConvertTo-PodeOASchemaObjectProperty {
         [hashtable[]]
         $Properties
     )
-
     $schema = @{}
-
     foreach ($prop in $Properties) {
-        $schema[$prop.name] = ($prop | ConvertTo-PodeOASchemaProperty -InObject)
+        if ( @('allOf', 'oneOf', 'anyOf') -icontains $prop.type  ) {
+            switch ($prop.type) {
+                'allof' { $prop.type = 'allOf' }
+                'oneof' { $prop.type = 'oneOf' }
+                'anyof' { $prop.type = 'anyOf' }
+            } 
```

Comment:
To be safe: `switch ($prop.type.ToLower())`; this way it matches the if-statement just above it.